.circleci/config.yml

```diff
@@ -6,6 +6,7 @@ install-python-components: &install-python-components
     pip3 install git+https://github.com/FEniCS/basix.git
     pip3 install git+https://github.com/FEniCS/ufl.git
     pip3 install git+https://github.com/FEniCS/ffcx.git
+    pip3 install sphinx==3.4.3
```

Comment:
What do we need sphinx for?

Reply:
Building the docs on the CI.

Sphinx was upgraded yesterday and containers have new version, but the new release has a bug (it's on the Sphinx issue tracker).